guppylang/definition/struct.py

```diff
@@ -90,16 +91,6 @@ class RawStructDef(TypeDef, ParsableDef):
     """A raw struct type definition that has not been parsed yet."""
 
     python_class: type
-    python_scope: PyScope = field(repr=False)
-
-    def __getitem__(self, item: Any) -> "RawStructDef":
```

Comment:
drive-by: This `__getitem__` should have been removed before when we added the comptime feature...